### PR TITLE
VBLOCKS-2139: Fixed audio routing

### PR DESCRIPTION
**Pull request by afalls-twilio**
## Submission Checklist

 - [x] Updated the `CHANGELOG.md` to reflect any **feature**, **bug fixes**, or **known issues** made in the source code
 - [x] Tested code changes and observed expected behavior in the example app
 - [ ] Performed a visual inspection of the `Files changed` tab prior to submitting the pull request for review to ensure proper usage of the style guide

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

## Description

Removed hard-coded behavior for audio routing and now utilizes audioswitch 

## Breakdown

- Updated to use a sound-pool
- Made notifications be 'silent' so that service can manage audio

## Validation

- Tested both out going and incoming calls

## Additional Notes

None

### CODE REVIEW

**Review by kbagchiGWC — APPROVED** [review]

File: android/src/main/java/com/twiliovoicereactnative/Constants.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 public class Constants {
   public static final String CALL_SID_KEY = "CALL_SID";
+  public static final String VOICE_CHANNEL_GROUP = "notificaton-group";
```
Comment (mhuynh5757) [inline android/src/main/java/com/twiliovoicereactnative/Constants.java:5]:
```suggestion
  public static final String VOICE_CHANNEL_GROUP = "notification-group";
```

File: android/src/main/java/com/twiliovoicereactnative/NotificationUtility.java [hunk 2]
```diff
@@ -293,41 +298,67 @@ public static Notification createWakeupAppNotification(String callSid, int notif
...
+  public static void destroyNotificationChannels(Context context) {
+    NotificationManager notificationManager =
+      (NotificationManager)context.getSystemService(Context.NOTIFICATION_SERVICE);
+    notificationManager.deleteNotificationChannelGroup(Constants.VOICE_CHANNEL_GROUP);
+  }
+
+  @TargetApi(Build.VERSION_CODES.O)
+  private static NotificationChannel createNotificationChannel(Context context,
+                                                               final String voiceChannelId) {
+    final int notificaitonImportance = getChannelImportance(voiceChannelId);
```
Comment (mhuynh5757) [inline android/src/main/java/com/twiliovoicereactnative/NotificationUtility.java:326]:
```suggestion
    final int notificationImportance = getChannelImportance(voiceChannelId);
```

**Review by mhuynh5757 — CHANGES_REQUESTED** [review]
Looks good, just a few typos.

File: android/src/main/java/com/twiliovoicereactnative/NotificationUtility.java (hunk 2 above)
Comment (mhuynh5757) [reply android/src/main/java/com/twiliovoicereactnative/NotificationUtility.java]: I think there are other references to this variable that are now invalid if we just merge this suggested change. @kbagchiGWC

**Review by mhuynh5757 — APPROVED** [review]

**Pull request merged**